roseau/load_flow/models/grounds.py

```diff
@@ -54,23 +68,27 @@ def connect(self, bus: Bus, phase: str = "n") -> None:
                 The phase of the connection. It must be one of ``{"a", "b", "c", "n"}`` and must be
                 present in the bus phases. Defaults to ``"n"``.
         """
+        if bus.id in self._connected_buses:
```

Comment:
I wrote it this way then changed it because it is almost certainly a mistake to connect the same ground to the same bus multiple times. I think it is better to raise an error explicitly and let the user know they used the wrong ground or wrong bus instead of silently passing.